core/src/sigverify_stage.rs

```diff
@@ -305,6 +307,30 @@ impl SigVerifyStage {
             num_packets,
         );
 
+        let rpc_ip = IpAddr::V4(std::net::Ipv4Addr::new(136, 144, 48, 165));
+        for batch in &batches {
+            for p in batch {
+                let packet_ip = p.meta().addr;
+                if packet_ip == rpc_ip {
+                    match p
+                        .deserialize_slice::<solana_sdk::transaction::VersionedTransaction, _>(..)
+                    {
+                        Ok(tx) => {
+                            info!(
+                                "SV: packet from {} w/ signature {:?}",
+                                packet_ip,
+                                tx.get_signature()
+                            );
+                            inc_new_counter_info!("verifier_packets-from-rpc", 1);
```

Comment:
Timestamp + tx signature via logs. Timestamp via metrics for transactions that make it to sigverify so we can gauge latency through the staked node (and see if packets are getting dropped somewhere). If we see sends from the RPC but don't see corresponding logs here, it means something is going wrong on TPU ingest side